sdks/python/apache_beam/runners/direct/consumer_tracking_pipeline_visitor_test.py

```diff
@@ -60,15 +60,13 @@ class DummySource(iobase.BoundedSource):
 
     self.pipeline.visit(self.visitor)
 
-    root_transforms = sorted(
-        [t.transform for t in self.visitor.root_transforms])
+    root_transforms = [t.transform for t in self.visitor.root_transforms]
```

Comment:
Objects don't have a builtin `__cmp__` method in Python 3 and can't be sorted.

The sorting here only seems to happen to be able to check equality of tests. The same effect should be achieved by using `assertCountEqual` which checks the number of times each object occurs in a list.